index.bs

```diff
@@ -84,6 +87,8 @@ The keygen element should be replaced by a new API better suited for modern day
 
 Additionally, any installation of a client certificate (e.g., the <code>application/x-x509-*-cert</code> mimetype) must also require user consent.
 
+Finaly - it may be desirable to give the user some level of control over the labeling of the certificate - i.e. how it is presented in the user interface. This is especially important when the user also uses smartcard or OS level certificates.
```

Comment:
I'd group this under the last bullet, I see it as a form of extensibility. This detail is nice to capture, so perhaps the above last bullet can be restructured into a title bullet and two sub-bullets.